python/1manifoldLearning.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'CSV Reader'
+pointCloudcsv = CSVReader(FileName=['pointCloud.csv'])
+
+# create a new 'TTK DimensionReduction'
+tTKDimensionReduction1 = TTKDimensionReduction(Input=pointCloudcsv, ModulePath='default')
+tTKDimensionReduction1.InputColumns = ['Points:0', 'Points:1', 'Points:2']
+tTKDimensionReduction1.UseAllCores = 0
+
+# create a new 'Table To Points'
+tableToPoints2 = TableToPoints(Input=tTKDimensionReduction1)
+tableToPoints2.XColumn = 'Component_0'
+tableToPoints2.YColumn = 'Component_1'
+tableToPoints2.ZColumn = 'Component_0'
+tableToPoints2.a2DPoints = 1
+
+# create a new 'Gaussian Resampling'
+gaussianResampling2 = GaussianResampling(Input=tableToPoints2)
+gaussianResampling2.ResampleField = ['POINTS', 'ignore arrays']
+gaussianResampling2.ResamplingGrid = [128, 64, 3]
+gaussianResampling2.SplatAccumulationMode = 'Sum'
+
+# create a new 'Table To Points'
+tableToPoints1 = TableToPoints(Input=pointCloudcsv)
+tableToPoints1.XColumn = 'Points:0'
+tableToPoints1.YColumn = 'Points:1'
+tableToPoints1.ZColumn = 'Points:2'
+
+# create a new 'Gaussian Resampling'
+gaussianResampling1 = GaussianResampling(Input=tableToPoints1)
+gaussianResampling1.ResampleField = ['POINTS', 'ignore arrays']
+gaussianResampling1.ResamplingGrid = [64, 64, 128]
+
+# create a new 'Outline'
+outline1 = Outline(Input=gaussianResampling1)
+
+# create a new 'Slice'
+slice1 = Slice(Input=gaussianResampling2)
+slice1.SliceType = 'Plane'
+slice1.HyperTreeGridSlicer = 'Plane'
+slice1.SliceOffsetValues = [0.0]
+
+# init the 'Plane' selected for 'SliceType'
+slice1.SliceType.Origin = [-0.00547350355230147, 0.0452015383244153, 0.0]
+slice1.SliceType.Normal = [0.0, 0.0, 1.0]
+
+# create a new 'TTK PersistenceDiagram'
+tTKPersistenceDiagram1 = TTKPersistenceDiagram(Input=slice1)
+tTKPersistenceDiagram1.ScalarField = ['POINTS', 'SplatterValues']
+tTKPersistenceDiagram1.InputOffsetField = [None, '']
+
+# create a new 'Threshold'
+threshold1 = Threshold(Input=tTKPersistenceDiagram1)
+threshold1.Scalars = ['CELLS', 'PairIdentifier']
+threshold1.ThresholdRange = [-0.1, 999.0]
+
+# create a new 'Threshold'
+persistenceThreshold = Threshold(Input=threshold1)
+persistenceThreshold.Scalars = ['CELLS', 'Persistence']
+persistenceThreshold.ThresholdRange = [3.0, 999.0]
+
+# create a new 'TTK TopologicalSimplification'
+tTKTopologicalSimplification1 = TTKTopologicalSimplification(Domain=slice1, Constraints=persistenceThreshold)
+tTKTopologicalSimplification1.ScalarField = ['POINTS', 'SplatterValues']
+tTKTopologicalSimplification1.InputOffsetField = [None, '']
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.